misc/docerina/src/main/java/org/ballerinalang/docgen/Generator.java

```diff
@@ -614,32 +619,16 @@ private static String getTypeName(BLangType bLangType) {
     private static String paramAnnotation(BLangNode node, BLangVariable param) {
         String subName = param.getName() == null ? param.type.tsymbol.name.value : param.getName().getValue();
 
-        if (node instanceof DocumentableNode && !((DocumentableNode) node).getDocumentationAttachments().isEmpty()) {
-            // new syntax
-            if (!((DocumentableNode) node).getDocumentationAttachments().isEmpty()) {
-                DocumentationNode bLangDocumentation = ((DocumentableNode) node).getDocumentationAttachments().get(0);
-                for (DocumentationAttributeNode attribute : bLangDocumentation.getAttributes()) {
-                    if (attribute instanceof BLangDocumentationAttribute) {
-                        BLangDocumentationAttribute docAttribute = (BLangDocumentationAttribute) attribute;
-                        if (docAttribute.docTag == DocTag.PARAM && docAttribute.getDocumentationField().toString()
-                                .equals(subName)) {
-                            return BallerinaDocUtils.mdToHtml(attribute.getDocumentationText());
-                        }
-                    }
-                }
-            }
-        } else {
-            for (AnnotationAttachmentNode annotation : getAnnotationAttachments(node)) {
-                BLangRecordLiteral bLangRecordLiteral = (BLangRecordLiteral) annotation.getExpression();
-                if (bLangRecordLiteral == null || bLangRecordLiteral.getKeyValuePairs().size() != 1) {
-                    continue;
-                }
-                BLangExpression bLangLiteral = bLangRecordLiteral.getKeyValuePairs().get(0).getValue();
-                String attribVal = bLangLiteral.toString();
-                if ((annotation.getAnnotationName().getValue().equals("Param")) && attribVal.startsWith(subName +
-                        ":")) {
-
-                    return attribVal.split(subName + ":")[1].trim();
+        if (node instanceof DocumentableNode
```

Comment:
This section of new code addition is repeating again in the same class. Lets move that to a method to avoid code duplication.